integration-tests/gatsby-source-wordpress/__tests__/index.js

```diff
@@ -97,12 +99,7 @@ describe(`[gatsby-source-wordpress] Run tests on develop build`, () => {
 
     return new Promise(resolve => {
       gatsbyDevelopProcess = spawnGatsbyProcess(`develop`)
-      gatsbyDevelopProcess.stdout.on("data", data => {
-        process.stdout.write(data)
-        if (data.toString().includes("http://localhost:8000")) {
-          resolve()
-        }
-      })
+      urling({ url: `http://localhost:8000/`, retry: 100 }).then(resolve)
```

Comment:
I had to do this because stdout wasn't being printed anymore for some reason when running tests locally. I changed the sdtio to inherit so I could see what was happening and debug - but that broke how this resolved.